.github/workflows/diff-explains.yml

```diff
@@ -0,0 +1,29 @@
+name: ChatGPT Reviewer
+
+permissions:
+  contents: read
+  pull-requests: write
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.repository }}-${{ github.event.number || github.head_ref || github.sha }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  chatgpt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+          ref: ${{github.event.pull_request.head.ref}}
+          submodules: false
+      - uses: fluxninja/chatgpt-pr-reviewer@main
+        env:
+          GITHUB_TOKEN: ${{ secrets.NOVUM_PRIVATE_REPOS }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+        with:
+          debug: true
+          review_comment_lgtm: true
```

Comment:
:robot: ChatGPT

This patch adds a GitHub Actions workflow to run ChatGPT PR Reviewer on pull requests. It sets up the necessary permissions and concurrency settings, and runs the `fluxninja/chatgpt-pr-reviewer` action with appropriate environment variables. The `debug` and `review_comment_lgtm` options are set to true. No issues were found during the review.

LGTM!

<!-- This is an auto-generated comment by ChatGPT -->